src/test/java/com/jitterted/mobreg/application/MemberFactory.java

```diff
@@ -1,20 +1,11 @@
 package com.jitterted.mobreg.application;
 
```

Comment:
Thanks for this cleanup. Since it no longer references anything in the `application` package, it can be moved to `domain`.